Note that console.trace() may include output from async sources (#24098)
mdn/content: files/en-us/web/api/console/trace/index.md

{{AvailableInWorkers}}

> > **Note:** In some browsers, `console.trace()` may also output the sequence of calls and asynchronous events leading to the current `console.trace()` which are not on the call stack — to help identify the origin of the current event evaluation loop.

Reviewer: ```suggestion
> **Note:** In some browsers, `console.trace()` may also output the sequence of calls and asynchronous events leading to the current `console.trace()` which are not on the call stack — to help identify the origin of the current event evaluation loop.
```

> > **Note:** In some browsers, `console.trace()` may also output the sequence of calls and asynchronous events leading to the current `console.trace()` which are  not on the call stack — to help identify the origin of the current event evaluation loop.

Reviewer: ```suggestion
> **Note:** In some browsers, `console.trace()` may also output the sequence of calls and asynchronous events leading to the current `console.trace()` which are not on the call stack — to help identify the origin of the current event evaluation loop.
```

> > **Note:** In some browsers, **`console.trace()`** may also include the sequence of calls and asynchronouse events that lead to the current **`console.trace()`** which are not not on the call stack to help identify the origin of the current event evaluation loop.

Reviewer: ```suggestion
> **Note:** In some browsers, `console.trace()` may also output the sequence of calls and asynchronous events leading to the current `console.trace()` which are  not on the call stack — to help identify the origin of the current event evaluation loop.
```
